manifests/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: preemptible-killer
+spec:
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
```

Comment:
AFAIR prometheus annotations are deprecated, it should be as service + servicemonitor.
I believe it's just a copy/paste from original file.

Reply:
They are, this file was rendered w/ helm template and its still the default on [master](https://github.com/estafette/estafette-gke-preemptible-killer/blob/master/chart/estafette-gke-preemptible-killer/values.yaml#L39)